schemas/content/1.0/schema.json

```diff
@@ -1186,6 +1188,49 @@
         "displayScore": {
             "type": "boolean"
         },
+        "catalogPaths" : {
+            "type": "array",
+            "items": {
+              "type": "object"
+            }
+          },
+          "competencies" : {
+            "type": "array",
+            "items": {
+              "type": "object"
+            }
+          },
+          "sharedWith" : {
+              "type": "array" 
+          },
+          "isExternal": {
+              "type": "boolean"
+          },
+          "learningMode": {
+              "type": "string",
+              "enum": [
+                "Self Paced",
+                "Instructor Led",
+                "Classroom"
+            ]
+          },
+          "creatorLogo" : {
+              "type": "string",
+              "format": "url"
+          },
+          "subTitles" : {
+              "type": "array",
+              "items": {
+                "type": "object"
+              }
+          },
+          "reviewStatus" : {
+              "type" : "string",
+              "enum": [
+                "InReview",
+                "Reviewed"
+            ]
+        },
```

Comment:
Where are we using this. If it is specific to a Primary Category, please add it in that specific primary category schema. Here we should have the properties which are generic.

These values also, let us have an explicit approval from DC before adding them here.

Reply:
Yes. The changes were reviewed by Rayulu. The approved PR is https://github.com/project-sunbird/knowledge-platform/pull/489. This is just the process of bringing in the PR to the current release